pkg/cmd/cli/cmd/newapp.go

```diff
@@ -612,7 +612,13 @@ func handleRunError(err error, baseName, commandName, commandPath string) error
 		transformError(err, baseName, commandName, commandPath, groups)
 	}
 	buf := &bytes.Buffer{}
+	fmt.Fprintf(buf, "Prior to API Object processing, here were the results from application argument classification:\n")
+	for key, value := range config.ArgumentClassificationErrors {
+		fmt.Fprintf(buf, fmt.Sprintf("\n%s:  %v\n", key, value))
+	}
+	fmt.Fprint(buf, "\n\nThen, the following errors were noted during API Object processing:\n\n")
```

Comment:
s/noted/occurred/ but this test will also have to be reworked to account for the possibility that it is the first output, once you make the change i suggested above.